### PR TITLE
added dev_langs and language identifier

### PR DESCRIPTION
**Pull request by mairaw**
Test PR for Bing search metadata

### CODE REVIEW

**Comment by BillWagner** [comment]
@mairaw I added the [WIP] tag to the title. The WIP bot doesn't appear to look at labels.

**Review by BillWagner — APPROVED** [review]
This LGTM. You can :shipit: when ready.

Is there anything we should be checking for search? Or will we need to wait for that until it's merged and published?

**Comment by mairaw** [comment]
This is something the Bing team has asked us to add to test something on their end.

**Pull request merged**
